src/terminal/TerminalClientMain.cpp

```diff
@@ -38,60 +38,65 @@ int main(int argc, char** argv) {
   // Parse command line arguments
   cxxopts::Options options("et", "Remote shell for the busy and impatient");
   try {
-    options.positional_help("[user@]hostname[:port]").show_positional_help();
     options.allow_unrecognised_options();
-
-    options.add_options()             //
-        ("h,help", "Print help")      //
-        ("version", "Print version")  //
-        ("u,username", "Username")    //
+    options.positional_help("");
+    options.custom_help("[OPTION...] [user@]host[:port]\n\n"
+        "  Note that 'host' can be a hostname or ipv4 address with or without a port\n"
+        "  or an ipv6 address. If the ipv6 address is abbreviated with :: then it must\n"
+        "  be specfied without a port (use -p,--port)."
+    );
+
+    options.add_options()
+        ("h,help", "Print help")
+        ("version", "Print version")
+        ("u,username", "Username")
         ("host", "Remote host name",
-         cxxopts::value<std::string>())  //
```

Comment:
I removed all these trailing `//` as I'm not sure why there are there.  It looks like they are supposed to be `\` like when you extend a shell command in a script but they are not necessary in this context.

Reply:
They are to avoid clang-format from putting multiple options per line